apiclient/harvester_api/managers.py

```diff
@@ -184,9 +184,9 @@ def get(self, name="", namespace=DEFAULT_NAMESPACE, *, raw=False):
         path = self.PATH_fmt.format(uid=f"/{name}", ns=namespace)
         return self._get(path, raw=raw)
 
-    def create(self, name, volume_spec, namespace=DEFAULT_NAMESPACE, *, raw=False):
+    def create(self, name, volume_spec, namespace=DEFAULT_NAMESPACE, *, raw=False, image_id=None):
```

Comment:
I thought we could let `image_id` be position argument would be more convenient:
```python
def create(self, name, volume_spec, namespace=DEFAULT_NAMESPACE, image_id=None, *, raw=False): ...
```

Reply:
Fixed.